.github/workflows/release.yaml

```diff
@@ -1,11 +1,15 @@
-name: Create ui-kit release artifact
+name: Create ui-kit release artifact and publish to ComfyUI registry
 
 on:
   push:
     branches:
```

Comment:
Will need to remove because commits need to sync to yondonfu/main before attempting to publish to registry

```suggestion
```